core/templates/components/state-directives/hint-editor/hint-editor.component.ts

```diff
@@ -71,7 +74,10 @@ export class HintEditorComponent implements OnInit, OnDestroy {
   }
 
   isHintLengthExceeded(): boolean {
-    return (this.hint.hintContent._html.length > 500);
+    return Boolean(
+      this.htmlLengthService.computeHtmlLength(
+        this.hint.hintContent._html, 'character') >
```

Comment:
This is still using a hardcoded string rather than the named constant.

Reply:
Done!